Package.swift

```diff
@@ -201,8 +201,5 @@ let package = Package(
         .testTarget(
             name: "ElasticComputeCloudClientTests",
             dependencies: ["ElasticComputeCloudClient"]),
-        .testTarget(
```

Comment:
I wouldn't think we'd want to remove these tests.

Reply:
Right, will add it back. Considering that this was code-generated, were these tests added manually?

Reply:
This target should have been added to the code generator but looks like it wasn't.